src/Orleans/Telemetry/Severity.cs

```diff
@@ -4,10 +4,11 @@
     public enum Severity
     {
         Off = 0,
-        Error = 1,
```

Comment:
it changed to have a better mapping to LogLevel. But yeah I think I should change it back. it's not worthwhile